src/test/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusinessIntegrationSpec.scala

```diff
@@ -18,46 +18,47 @@ class AgoraBusinessIntegrationSpec extends FlatSpec with BeforeAndAfterAll with
   }
 
   "Agora" should "be able to store a task configuration with a valid (extant) official docker image" in {
-    val entity = agoraBusiness.insert(testAgoraEntityWithValidOfficialDockerImageInWdl, mockAutheticatedOwner.get)
-    assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAutheticatedOwner.get) === entity)
+    val entity = agoraBusiness.insert(testAgoraEntityWithValidOfficialDockerImageInWdl, mockAuthenticatedOwner.get)
+    val actual = agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAuthenticatedOwner.get)
+    assert(actual === entity.addManagers(Seq(mockAuthenticatedOwner.get)))
```

Comment:
for code style, I'd prefer to do all the manipulation in one place, and keep the assert simple:
```
val entity = agoraBusiness.insert(testAgoraEntityWithValidOfficialDockerImageInWdl, mockAuthenticatedOwner.get)
  .addManagers(Seq(mockAuthenticatedOwner.get))
val actual = agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAuthenticatedOwner.get)
assert(actual === entity)
```

(untested code, but you get the gist)

also adjust the test at line 42